spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
I think this comment should be removed. I don't want to start documenting these sorts of dependencies in an ad-hoc way using comments. If it's important to do let's figure out a better way.